README.md

```diff
@@ -1,9 +1,9 @@
 # scalajs-java-logging
 
 [![Build Status](https://travis-ci.org/scala-js/scala-js-java-logging.svg?branch=master)](https://travis-ci.org/scala-js/scala-js-java-logging)
-[![Scala.js](https://www.scala-js.org/assets/badges/scalajs-0.6.9.svg)](https://www.scala-js.org/)
+[![Scala.js](https://www.scala-js.org/assets/badges/scalajs-0.6.8.svg)](https://www.scala-js.org/)
```

Comment:
@cquiroz You are actually right to change it to 0.6.8. But ideally that would be in a separate PR, as it irrelevant to this PR.